src/routes/users.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const express = require('express');
+const {
+  getAllAction,
+  addUsersAction,
```

Comment:
The function name should be 'addUserAction' instead of 'addUsersAction' for consistency.